### ISSUE TITLE
How can i reference 1.4.0.BUILD-SNAPSHOT in my pom.xml ?

### ISSUE THREAD

**Issue by arcseldon** [spring-projects/spring-boot#5974]
All your samples reference ultimately reference version `1.4.0.BUILD-SNAPSHOT`

Even your documentation makes statements like `sdkman` and `maven` can be used - eg. see [here](http://docs.spring.io/spring-boot/docs/1.4.0.BUILD-SNAPSHOT/reference/htmlsingle/#getting-started-maven-installation)

However, locally I simply cannot find this version either available via `sdkman` or `maven`.

I want to reference as follows in my `pom.xml` - please can you advise how I can get this to work?

```
<parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
        <version>1.4.0.BUILD-SNAPSHOT</version>
    </parent>
```

I was able to download the `.tar.gz` bundle and have:

```
$  spring --version
Spring CLI v1.4.0.BUILD-SNAPSHOT
```

for the cli off the command line only.

Was following the samples, and there are 1.4.0 specific features I wish to use.

**Comment by arcseldon** [comment]
Ok, so finally by looking through the github source code, I found this up the `pom.xml` hierarchy:

```
 <repositories>
        <!-- Repositories to allow snapshot and milestone BOM imports during
            development. This section is stripped out when a full release is prepared. -->
        <repository>
            <id>spring-milestones</id>
            <name>Spring Milestones</name>
            <url>http://repo.spring.io/milestone</url>
            <snapshots>
                <enabled>false</enabled>
            </snapshots>
        </repository>
        <repository>
            <id>spring-snapshots</id>
            <name>Spring Snapshots</name>
            <url>http://repo.spring.io/snapshot</url>
            <snapshots>
                <enabled>true</enabled>
            </snapshots>
        </repository>
    </repositories>
```

Problem solved - PLEASE add to your documentation page :)

**Comment by wilkinsona** [comment]
@arcseldon What documentation page were you looking at? http://projects.spring.io/spring-boot shows the necessary repositories based on the selected version. http://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/htmlsingle/#getting-started-maven-installation also shows the repositories. I've just noticed that this is the same documentation that you linked to above. What were you expecting to see?

**Comment by arcseldon** [comment]
@wilkinsona - Thank you very much for the reply, and I concede your first link does indeed have some hidden drop-downs too - actually very convenient now I realise it's there but I did miss it.

I checked first `Maven Central` which only went as far as 1.3.5. Next, I read this here about [sdkman](http://docs.spring.io/spring-boot/docs/1.4.0.BUILD-SNAPSHOT/reference/htmlsingle/#getting-started-sdkman-cli-installation)  But when I reference available versions in sdkman it is not published.. (think i used something like `sdk list spring`).  

All the same, thanks again and problem solved!
